foundations/javascript_basics/javascript_developer_tools.md

```diff
@@ -34,15 +33,15 @@ There are three ways to open the Developer Tools menu:
 Google has updated some of the required sections in the below tutorials and some elements have changed or no longer exist, but you can still follow along using the same functionality and tools that they cover. For example, you may be asked to inspect a button that is no longer on the page- you can still follow along and inspect existing elements without issue.
 
 1. Head to the [Chrome DevTools Documentation](https://developers.google.com/web/tools/chrome-devtools/) by Google. The following subsections cover what you'll be using the Developer Tools for 95% of the time.  Feel free to skip the elements you are already familiar with:
```

Comment:
The chrome developer tools link redirects to a different URL. We should update the current link to https://developer.chrome.com/docs/devtools/

```suggestion
1. Head to the [Chrome DevTools Documentation](https://developer.chrome.com/docs/devtools/) by Google. The following subsections cover what you'll be using the Developer Tools for 95% of the time.  Feel free to skip the elements you are already familiar with:
```

Reply:
Update as requested